dubbo-rpc/dubbo-rpc-dubbo/pom.xml

```diff
@@ -60,6 +60,12 @@
             <version>${project.parent.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
It seems that only adding dependencies does not make the test cover mina.
We still need some modifications to the UT. Both the server and the client have a cache. The cached key is the address (127.0.0.1:9000). In order to create a different Client or Server instance, we need to modify the port number of the UT.

Reply:
Personally think that different port numbers are better.

Reply:
Just to be clear, should I make the change to make every test have a unique port number in the URL before the PR can be merged? Right now, the PR only concerns testDubboProtocolWithMina, should changing port numbers for all other tests be put in a separate PR?

Reply:
Since most of the ut is stable, I think it is not necessary to change every port.